opentracing-spring-web/src/test/java/io/opentracing/contrib/spring/web/interceptor/TracingHandlerInterceptorJavaConfigIntegrationTest.java

```diff
@@ -0,0 +1,41 @@
+package io.opentracing.contrib.spring.web.interceptor;
+
+import io.opentracing.Tracer;
+import io.opentracing.mock.MockTracer;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = TracingHandlerInterceptorJavaConfigIntegrationTest.TestConfiguration.class)
+public class TracingHandlerInterceptorJavaConfigIntegrationTest {
+
+    @Autowired
+    private TracingHandlerInterceptor interceptor;
+
+    @Test
+    public void testAutowired() {
+        assertNotNull(interceptor);
```

Comment:
What does this test actually do?

Reply:
The same thing that the existing XML configuration test does. It ensures that the `TracingHandlerInterceptor` can be created correctly when using Java Config (as the XML test did for the XML configuration)